python/tests/unittests/test_SpinOperator.py

```diff
@@ -148,12 +257,17 @@ def test_spin_op_vqe():
     # Checking equality operators.
     assert spin.x(2) != hamiltonian
     assert hamiltonian == hamiltonian
-    # Checking the indexing operator.
-    assert hamiltonian[1].to_string() == "(-2.1433,-0) X0X1"
     assert hamiltonian.get_term_count() == 5
-    want_string = "(5.907,0) I0I1 + (-2.1433,-0) X0X1 + (-2.1433,-0) Y0Y1 + (0.21829,0) Z0I1 + (-6.125,-0) I0Z1"
-    assert want_string == hamiltonian.to_string()
-    assert want_string == str(hamiltonian)
+
+    data, coeffs = hamiltonian.get_raw_data()
+    assert (len(data) == 5)
+    assert (len(data[0]) == 4)
+    expected = [[0,0,0,0], [1, 1, 0, 0], [1,1,1,1], [0,0,0,1], [0,0,1,0]]
+    assert (all([d in expected for d in data]))
+    expected = [5.907, -2.1433, -2.1433, .21829, -6.125]
```

Comment:
```suggestion
    want_data = [5.907, -2.1433, -2.1433, .21829, -6.125]
```
And swap out below assert with `assert np.allclose(`